packages/cms-cli/commands/fetch.js

```diff
@@ -72,6 +73,7 @@ const builder = yargs => {
   addPortalOptions(yargs, true);
   addOverwriteOptions(yargs, true);
   addModeOptions(yargs, { read: true }, true);
+  addUseEnvironmentOptions(yargs, true);
```

Comment:
Is this being used in every command, or just some?  If it is, we can probably add it more as a middleware like we do with the logger options for yargs

Reply:
Not all, but most